Resources/Prototypes/_RMC14/Entities/Objects/Misc/clipboard.yml

```
@@ -55,3 +55,62 @@
     blacklist:
       components:
       - Xeno
+
+- type: entity
+  parent: CMClipboard
+  id: RMCWeYaClipboard
+  name: Weston-Yamada clipboard
+  description: A branded clipboard, perfect for keeping your forms in order.
+  components:
+  - type: Sprite
+    sprite: _RMC14/Objects/Misc/weya_clipboard.rsi
+    layers:
+    - state: clipboard
+    - state: clipboard_paper
+      map: [ "clipboard_paper" ]
+      visible: false
+    - state: clipboard_pen
+      map: [ "clipboard_pen" ]
+      visible: false
+    - state: clipboard_over
+  - type: Item
+    sprite: _RMC14/Objects/Misc/weya_clipboard.rsi
+    size: Small
+  - type: Clothing
+    slots: [belt]
+    quickEquip: false
+    sprite: _RMC14/Objects/Misc/weya_clipboard.rsi
+  - type: ItemMapper
+    mapLayers:
+      clipboard_paper:
+        whitelist:
+          tags:
+          - Document
+      clipboard_pen:
+        whitelist:
+          tags:
+          - Write
+  - type: ItemSlots
+    slots:
+      pen_slot:
+        name: clipboard-slot-component-slot-name-pen
+        whitelist:
+          tags:
+          - Write
+        insertOnInteract: true
+  - type: StorageFill
+    contents:
+    - id: CMPaper
+      prob: 0.5
+    - id: CMPaper
+      prob: 0.4
+    - id: CMPaper
+      prob: 0.3
+    - id: CMPaper
+      prob: 0.2
+    - id: CMPaper
+      prob: 0.2
```

```suggestion
  - type: StorageFill
    contents:
    - id: CMPaper
      amount: 4
```
If this is not a survivor item just let them have the paper